cpp/src/reductions/scan/rank_scan.cu

```diff
@@ -105,11 +105,31 @@ std::unique_ptr<column> inclusive_rank_scan(column_view const& order_by,
   return rank_generator(
     order_by,
     has_nested_nulls(table_view{{order_by}}),
-    [] __device__(bool equality, auto row_index) { return equality ? row_index + 1 : 0; },
+    [] __device__(bool unequal, auto row_index) { return unequal ? row_index + 1 : 0; },
     DeviceMax{},
     stream,
     mr);
 }
 
+std::unique_ptr<column> inclusive_percent_rank_scan(column_view const& order_by,
+                                                    rmm::cuda_stream_view stream,
+                                                    rmm::mr::device_memory_resource* mr)
+{
+  auto const rank_column =
+    inclusive_rank_scan(order_by, stream, rmm::mr::get_current_device_resource());
+  auto const rank          = rank_column->view();
```

Comment:
Suggest to rename this as it is confusing.
```suggestion
  auto const rank_view          = rank_column->view();
```

Reply:
I didn't see what the problem was on the first reading: I'm using `rank` for both the column-view and the lambda's parameter.

Will fix.

Reply:
Fixed.